### PR TITLE
Split liquibase changeset of HTextFlowTarget and History

### PR DESCRIPTION
**Pull request by aeng**
https://bugzilla.redhat.com/show_bug.cgi?id=1207980

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="http://app.review.ninja/zanata/zanata-server/pull/827" target="_blank">ReviewNinja

File: zanata-war/src/main/resources/db/changelogs/db.changelog-3.7.xml [hunk 1]
```diff
@@ -74,13 +74,27 @@
...
     </addColumn>
+  </changeSet>
+
+  <changeSet id="4" author="aeng@redhat.com">
+    <preConditions onFail="MARK_RAN">
+      <not>
+        <changeSetExecuted id="3" author="aeng@redhat.com" changeLogFile="db/changelogs/db.changelog-3.7.xml"/>
+      </not>
+    </preConditions>
+    <comment>Add revisionComment HTextFlowTargetHistory</comment>
```
Comment (seanf) [inline zanata-war/src/main/resources/db/changelogs/db.changelog-3.7.xml]: You accidentally a word.

**Comment by seanf** [comment]
:+1:

**Comment by djansen-redhat** [comment]
:x: Cannot merge

**Comment by aeng** [comment]
release notes...

**Comment by definite** [comment]
@zanata-jenkins retest this please

**Comment by definite** [comment]
Error log
http://paste.fedoraproject.org/226422/32802377/

**Comment by seanf** [comment]
@definite Did you increase the deployment timeout as in the release notes?

Mysqld should eventually have finished off the DBML operation it was on.  (I usually watch the I/O stats to see when that is, but I think you should be able to tell from mysql's log file.)  You'll need to remove the dbchangeloglock, but the singular changeset should make the recovery a lot simpler than it would be otherwise.  Zanata should then be able to pick up at the next changeset.

**Comment by aeng** [comment]
@definite I think the imported database has some record in DATABASECHANGELOGLOCK. If you've only run once as suggested in the log, then, remove all record in DATABASECHANGELOGLOCK table and startup Zanata again.

**Comment by definite** [comment]
@zanata-jenkins  retest this please

**Comment by definite** [comment]
16:35:25,825 INFO  [liquibase] (ServerService Thread Pool -- 63) db/db.changelog.xml: db/changelogs/db.changelog-3.7.xml::3-split::aeng@redhat.com: ChangeSet db/changelogs/db.changelog-3.7.xml::3-split::aeng@redhat.com ran successfully in 4235350ms

More than 70 mins on my machine.

**Comment by aeng** [comment]
Merged into integration/master. https://github.com/zanata/zanata-server/commit/6c19620905205d3cd4ad85f25e9a946301f85fed

**Pull request closed without merging**